lib/rules/no-v-html.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @fileoverview Restrict or warn use of v-html to prevent XSS attack
+ * @author Nathan Zeplowitz
+ */
+'use strict'
+const utils = require('../utils')
+
+// ------------------------------------------------------------------------------
+// Rule Definitionutilu
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'disallow use of v-html to prevent XSS attack',
+      category: 'security enhancement',
```

Comment:
Please use `undefined` as category. We're going to assign categories in major releases, as it results in automatic generation of sharable configs.

Reply:
I've updated the PR with this change.